sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -28,9 +28,11 @@ pip install azure-ai-textanalytics --pre
 Text Analytics supports both [multi-service and single-service access](https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows). 
 Create a Cognitive Services resource if you plan to access multiple cognitive services under a single endpoint/key. For Text Analytics access only, create a Text Analytics resource.
 
-You can create either resource using the
-[Azure Portal](https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows#create-a-new-azure-cognitive-services-resource)
-or [Azure CLI](https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account-cli?tabs=windows).
+You can create the resource using
+
+**Option 1:** [Azure Portal](https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesTextAnalytics)
```

Comment:
nit: better to put the link at the end of file. So the readme format is nicer and easier to maintain.

Reply:
Yes, thanks for pointing it out. I do plan to do this, but prioritizing other things right now